pkg/serviceimport/map_test.go

```diff
@@ -84,6 +147,50 @@ var _ = Describe("ServiceImport Map", func() {
 				Expect(getIP(namespace1, service1)).To(Equal(secondIP))
 			}
```

Comment:
This can be replaced by `testRoundRobin(namespace1, service1, "", "", []string{serviceIP1, serviceIP2})`